WebService/v1/Models/SimulationApiModel/SimulationApiModel.cs

```diff
@@ -165,6 +169,7 @@ public Simulation ToServiceModel(string id = "")
 
             result.DeviceModels = SimulationDeviceModelRef.FromServiceModel(value.DeviceModels);
             result.Statistics = SimulationStatistics.FromServiceModel(value.Statistics);
+            result.RateLimits = SimulationRateLimits.FromServiceModel(value.RateLimits, rateReporter);
```

Comment:
`rateReporter` seems a strange var name, could you rename it to match the interface, e.g. `rateLimits` ?

Reply:
updated